### PR TITLE
Book Store: Add components

### PR DESCRIPTION
**Pull request by ernestmusong**
## In my bookstore app I have added the following:

- Created a directory for components
- Added two routes
- Added a navigation component with links to the two routes

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -2,7 +2,34 @@ import React from 'react';
...
+          </div>
+          <div className="chapter">
+            <h3>current chapter</h3>
+            <h3>chapter 17</h3>
+            <button type="button">update progress</button>
+          </div>
+        </div>
+      </div>
+    </li>
   );
```
Comment (abdulhamiid) [inline src/components/Book.js]: To make the `<Book>` component reusable, please kindly pass the components' data as `Props`. and render the values inside the component. 👍

`Props` is short for properties in ReactJS & they are used to pass data between React components.
e.g 
```JavaScript
function Book({
  title, author
}){
  return (
  . . .
   )
}
```

File: src/components/Books.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
 import React from 'react';
+import Book from './Book';
 
 function Books() {
   return (
-    <div>Books</div>
+    <ul>
+      <Book />
+    </ul>
```
Comment (abdulhamiid) [inline src/components/Books.js]: Please let's pass the book data as a prop and renders the title and the author. e.g

```JavaScript
<>
  <Book title="The Hunger Games" author="Suzanne Collins" />
  <Book title="Capital in the Twenty-First Century" author="Suzanne Collins" />
</>
```

This way, we can reuse the `<Book>` components, by changing the `Props` values. 💪

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @ernestmusong 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use Git Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abdulhamiid** [comment]
Please kindly remove/update the Live-demo link so that it navigates to the correct URL

```
## 🚀 Live Demo <a name="live-demo"></a>

- [Live Demo Link](https://deploy-preview-5--musong-math-magicians.netlify.app/)
```

**Comment by ernestmusong** [comment]
Thanks @abdulhamiid for your feedback. I have now added the tiltle and author values as props.

File: README.md [hunk 3]
```diff
@@ -45,7 +45,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://deploy-preview-5--musong-math-magicians.netlify.app/)
+- [Live Demo Link](#)
```
Comment (TadesseDev) [inline README.md:49]: - [Optional] Dear @ernestmusong, I see you miss understood the requested changes by the previous reviewer. Please for the live version, you should delete the live demo section if you decide not to deploy the application. If project requirements say you should deploy, then you should put the link to the live version `NOT TO YOUR GITHUB REPO`.

**Review by TadesseDev — APPROVED** [review]
### Hi @ernestmusong 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**